ui/lib/pki/addon/components/pki-issuer-cross-sign.js

```diff
@@ -73,6 +75,23 @@ export default class PkiIssuerCrossSign extends Component {
   *submit(e) {
     e.preventDefault();
     this.signedIssuers = [];
+    this.validationErrors = [];
+
+    // Validate name input for new issuer does not already exist in mount
+    for (let row = 0; row < this.formData.length; row++) {
+      const { intermediateMount, newCrossSignedIssuer } = this.formData[row];
+      const issuers = yield this.store
+        .query('pki/issuer', { backend: intermediateMount })
+        .then((resp) => resp.map(({ issuerName, issuerId }) => ({ issuerName, issuerId })))
+        .catch(() => []);
```

Comment:
If there's a permission error, the API can handle and return later in the cross-signing process